packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -782,4 +784,43 @@ export class PhoenixObjects {
 
     return object;
   }
+
+  /**
+   * Type for drawing irregular calorimeter cell comprising 8 xyz vertices in arbitrary geometry.
+   * @param layer Calorimeter layer
```

Comment:
Since the function parameter is an object, all `@param` docs can be written with the `irrCells.` prefix.

```ts
/**
   * Type for drawing irregular calorimeter cell comprising 8 xyz vertices in arbitrary geometry.
   * @param irrCells Properties of irregular calo cell.
   * @param irrCells.layer Calorimeter layer
   * @param irrCells.vtx Flattened list of 8 vertex coordinates (24 floats)
   * @param irrCells.color [R,G,B] integer list
   * @param irrCells.opacity value from 0 to 1
   * @returns the cell
   */
```

Reference: https://jsdoc.app/tags-param.html#parameters-with-properties